app/controllers/audience_controller.rb

```diff
@@ -1,16 +1,31 @@
 # frozen_string_literal: true
 
 class AudienceController < Sellers::BaseController
-  before_action :set_body_id_as_app
-  before_action :set_time_range, only: %i[data_by_date]
+  before_action :set_time_range, only: :index
 
   after_action :set_dashboard_preference_to_audience, only: :index
   before_action :check_payment_details, only: :index
 
+  layout "inertia", only: :index
+
   def index
     authorize :audience
 
-    @total_follower_count = current_seller.audience_members.where(follower: true).count
+    total_follower_count = current_seller.audience_members.where(follower: true).count
+
+    render inertia: "Audience/Index", props: {
```

Comment:
(Nit, nothing major) Is there a specific reason to keep it at the top level like this? It feels confusing a bit as I expect this to render at `/audience`

Maybe we should structure it to match with the route since sister projects follow a similar pattern. It might make it easier for contributors to navigate through the projects, wdyt?

```suggestion
    render inertia: "Dashboard/Audience/Index", props: {
```

Reply:
> but we would need to restructure the entire dashboard pages consistenty as a separate effort in my opinion, provided this would relevant to the users.

Makes sense, feels low ROI the way you put it 👍

 >  It would make sense to me to change the route to /analytics/audience, as it’s a little odd to have an analytics layout that several unrelated-sounding paths use, but not worth breaking people’s bookmarks to do that IMO but not worth breaking people’s bookmarks to do that IMO
 
Hmmm, we can add re-direction routes later on anyway from '/dashboard/audience` to `/analytics/audience` if the need arises. 

<hr />

Everything looks fine on my end from local testing and the failing specs also seem to be unrelated. 

For me it's good to go, the above comment is non-blocking 🚀